include/fi_util.h

```diff
@@ -65,6 +65,8 @@
 #include "rbtree.h"
 
 #define UTIL_FLAG_ERROR	(1ULL << 60)
+	
+#define OFI_CNTR_ENABLED	(1 << 0)
```

Comment:
ep.flags is at least used by the mlx provider and assigned to info->mode.  Let's bump this to 1ULL << 61 just to be safe.  Flag bits 60-63 are reserved as provider specific.  Other than that, the changes look fine to me.

Reply:
Oooh, Sure. Thank you for catching this.
The `OFI_CNTR_ENABLED` constant has been update to `1ULL << 61` value